sound/sound_library.h

```diff
@@ -9,39 +9,38 @@ enum SayType {
 };
 
 #ifdef SAY_COLOR_LIST
-EFFECT(clrlst); // spoken color names for SAY_COLOR_LIST
+  EFFECT(clrlst); // spoken color names for SAY_COLOR_LIST
 #endif
-// New colors should be added at end of enum and assigned numbers for each COLOR_ should not be changed.
-enum ColorNumber {
-  COLOR_RED = 1,
-  COLOR_ORANGERED = 2,
-  COLOR_DARKORANGE = 3,
-  COLOR_ORANGE = 4,
-  COLOR_GOLD = 5,
-  COLOR_YELLOW = 6,
-  COLOR_GREENYELLOW = 7,
-  COLOR_GREEN = 8,
-  COLOR_AQUAMARINE = 9,
-  COLOR_CYAN = 10,
-  COLOR_DEEPSKYBLUE = 11,
-  COLOR_DODGERBLUE = 12,
-  COLOR_BLUE = 13,
-  COLOR_ICEBLUE = 14,
-  COLOR_INDIGO = 15,
-  COLOR_PURPLE = 16,
-  COLOR_DEEPPURPLE = 17,
-  COLOR_MAGENTA = 18,
-  COLOR_DEEPPINK = 19,
-  COLOR_SILVER = 20,
-  COLOR_GLACIER = 21,
-  COLOR_ICEWHITE = 22,
-  COLOR_LIGHTCYAN = 23,   
-  COLOR_MOCCASIN = 24,
-  COLOR_LEMONCHIFFON = 25,
-  COLOR_NAVAJOWHITE = 26,
-  COLOR_WHITE = 27,
-};
-
+  // New colors should be added at end of enum and assigned numbers for each COLOR_ should not be changed.
+  enum ColorNumber {
```

Comment:
What is this doing in this PR?
This is never going to get merged if you keep adding more and more to it.

Reply:
You're right, too much at once. It's part of sound library use for speak blade ID. Related, but not directly involved. should have waited.
if it's ok, i'll close submit this less haphazardly, now that i understand what's actually going on.(mostly)